### PR TITLE
neptune import bug fixed

### PR DESCRIPTION
**Pull request by buus2**

### CODE REVIEW

File: minerva/backend/models/pytorch/callbacks.py [hunk 1]
```diff
@@ -333,6 +332,7 @@ def on_batch_end(self, *args, **kwargs):
 class NeptuneMonitor(Callback):
     def __init__(self):
         super().__init__()
+        from deepsense import neptune
```
Comment (kamil-kaczmarek) [inline minerva/backend/models/pytorch/callbacks.py]: @buus2 @jakubczakon 
In principle, I understand the idea, however, PEP8 clearly states that: 

> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.

 ( https://www.python.org/dev/peps/pep-0008/#imports )

We can accept it temporarily, but I think that we should figure out something better.

Comment (buus2) [reply minerva/backend/models/pytorch/callbacks.py]: agree

Comment (kamil-kaczmarek) [reply minerva/backend/models/pytorch/callbacks.py]: @buus2 @jakubczakon 

I have this simple idea to move all NeptuneMonitors to the separate file. So, we will have `.../callbacks.py` and `.../callbacks_neptune.py`. Two consequences:
1. refactor some code
2. direct distinction between _neptune_ and _no_neptune_ callbacks. If you do not use Neptune, you simply do not import anything from `.../callbacks_neptune.py`.

Comment (buus2) [reply minerva/backend/models/pytorch/callbacks.py]: good idea

**Pull request merged**